backend/services/theme_service.py

```
@@ -158,17 +166,11 @@ async def generate_with_face_reference(
                 "seed": random_seed,  # 🎲 FORCE NEW GENERATION: Prevents RunWare caching
-                "ipAdapters": [
-                    {
-                        "model": "runware:105@1",  # IP-Adapter FaceID model from documentation
-                        "guideImage": face_data_uri,  # Direct base64 data URI
-                        "weight": clamped_weight  # 🔧 CONFIGURABLE: Clamped between 0.0-1.0
-                    }
-                ]
             }
             
-            logger.info(f"🎯 RunWare IP-Adapter FaceID generation starting...")
+            logger.info(f"🎯 RunWare Image-to-Image generation starting...")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)